src/services/collections/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+
+import { collections } from "../../mock/collections";
+
+export const getCollectionsByUserID = (user_id) => {
+  const response = collections.filter((collect) => collect.user_id === user_id);
+  return response;
+};
+
+export const getCollectByID = (collect_id) => {
+  const response = collections.filter(
+    (collect) => collect.collection_id === collect_id
+  );
+  return response[0];
```

Comment:
Nesse caso você poderia usar o `find` ao invés do `filter`, assim você não iria precisar pegar a posição zero do array o que inclusive pode gerar um erro caso não seja encontrado nenhuma coleta com esse id

```suggestion
  const response = collections.find(
    (collect) => collect.collection_id === collect_id
  );
  return response;
```